packages/devtools/client/components/ComponentDetails.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import type { Component } from '@nuxt/schema'
+import type { Component } from 'nuxt/schema'
```

Comment:
Would be nice to be able to have the auto import for this

Reply:
This "Component" is different from vue's "Component". Types in `nuxt/schema` are mostly for module authors, and in modules, we recommended explicitly importing everything. I guess it's better to keep it explicit.